includes/utilities.inc

```diff
@@ -94,7 +97,7 @@ function islandora_iiif_manifests_createDefaultCanvas($canvas_object) {
     ),
     'images' => array(
       array(
-        '@id' => $canvas_object['service_uri'],
+        '@id' => $canvas_object['id_uri'],
```

Comment:
Are you sure that key 'id_uri' exists?

Reply:
Yes it does, here: https://github.com/bencomp/islandora_iiif_manifests/blob/1a4ee4a3e3c8482145a7a5351cb32718aaea5f3d/includes/utilities.inc#L358